README.md

```diff
@@ -124,12 +124,39 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object, setting as many fields on it as possible; the more fields that are set, the less likely it is that a user will be be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
 
 ```swift
```

Comment:
Should there be an obj c version of this as well?

Reply:
It's the same process conceptually for both languages; do you mean adding ObjC Code snippets? The whole README is in Swift as of now.

Reply:
My guess is that anyone who uses Obj C will be able to translate the Swift snippets themselves. So I'd say we're good with just having Swift in the README.